roles/common/tasks/idp_certs.yml

```diff
@@ -34,7 +49,7 @@
   ignore_errors: true
   register: cert_imported
   when:
-    - fips_enabled|bool
+    - create_bouncy_castle_keystore|bool
```

Comment:
Since these are both true simultaneously is this change made for more clarity while reading or any other reasons

Reply:
for non-kafka components, this create_bouncy_castle is always false since we support fips in kafka only. If we'll use fips, this task will also get triggered for non kafka components, which is wrong

Reply:
got it